make/winx86/bash_profile

```diff
@@ -14,21 +14,21 @@ QT_MINGWVER="mingw492_32"
 
 test_and_add_to_path ()
 {
-  if which $2 >/dev/null 2>&1 ; then
-    return
-  fi
+#  if which $2 >/dev/null 2>&1 ; then
+#    return
+#  fi
 
   if [ -f "$1/$2" ] ; then
     PATH=$1\:$PATH
   fi
 }
 
-test_and_add_to_path "$QT_BASEDIR/$QT_VERSION/$QT_MINGWVER/bin" "QtWebProcess.exe"
-test_and_add_to_path "$QT_BASEDIR/Tools/$QT_MINGWVER/bin" "gcc.exe"
 test_and_add_to_path "/C/Python27" "python.exe"
 test_and_add_to_path "/C/OpenOCD/0.4.0/bin/" "openocd.exe"
 test_and_add_to_path "/C/Program Files/NSIS/Unicode" "makensis.exe"
 test_and_add_to_path "/C/Program Files (x86)/NSIS/Unicode" "makensis.exe"
+test_and_add_to_path "$QT_BASEDIR/$QT_VERSION/$QT_MINGWVER/bin" "QtWebProcess.exe"
+test_and_add_to_path "$QT_BASEDIR/Tools/$QT_MINGWVER/bin" "gcc.exe"
```

Comment:
But they do-- the reason why this was opened is because Anaconda (python distribution) includes qmake 4.xx tools.